### ISSUE TITLE
CLI : FEATURE export filename contains double specifications text

### ISSUE THREAD

**Issue by Jedi121ch195** [CloudCompare/CloudCompare#2023]
Using CLI with this command :
cloudcompare.exe -SILENT -O -GLOBAL_SHIFT AUTO tranche_exemple_1.las -NO_TIMESTAMP -C_EXPORT_FMT LAS -FEATURE PLANARITY 0.8
Export filename is :
tranche_exemple_1_PLANARITY_FEATURE_KERNEL_0.8_PLANARITY_FEATURE_KERNEL_0.8.las
where _PLANARITY_FEATURE_KERNEL_0.8 is repeated twice

**Expected behaviour**
Export filenames should be :
tranche_exemple_1_PLANARITY_FEATURE_KERNEL_0.8.las

**Additional context**
Same behaviour if changing the FEATURE from PLANARITY to LINEARITY

**Your environment**
 - CC Version (or git hash/tag): CloudCompare_v2.13.1_bin_x64
 - OS & Version: Windows 10.0.19045
 - Graphics card: 
 - Qt Version (if compiling):

**Comment by Jedi121ch195** [comment]
Maybe here in /ccCommandLineCommands.cpp :
```
	QString fileNameExt = QObject::tr("%1_FEATURE_KERNEL_%2").arg(featureTypeStr).arg(kernelSize);
	for (size_t i = 0; i < cmd.clouds().size(); ++i)
	{
		CLCloudDesc& desc = cmd.clouds()[i];
		entities[i] = desc.pc;
		desc.basename += "_" + fileNameExt;
		entities[i]->setName(entities[i]->getName() + "_" + fileNameExt);
	}
```
where desc.basename is set then entities[i] name is also set depending on how autosave works.

**Comment by dgirardeau** [comment]
Sorry I was travelling. I'll try to take a look at this ASAP...

**Comment by dgirardeau** [comment]
Ok, this is now fixed by 7587f079f614cfda3f5947bd704155ff8dd69bcd

You can test the preview 2.13.2 version here: https://www.cloudcompare.org/release/CloudCompare_v2.13.2.preview_setup_x64.exe

Thanks for the feedback!
